webview/src/setup/components/dvc/CliUnavailable.tsx

```diff
@@ -33,7 +44,13 @@ export const CliUnavailable: React.FC<PropsWithChildren> = ({ children }) => {
     <EmptyState isFullScreen={false}>
       <h1>DVC is currently unavailable</h1>
       {children}
-      {contents}
+      <p>
+        New to DVC? Check out <a href="https://dvc.org/">dvc.org</a> to learn
+        more or our{' '}
```

Comment:
Because "to learn more" was added after the link, it feels like a verb is missing in front of "our demo".  Maybe rephrase the second part to be " or quickly try the extension with our demo"?

Reply:
Good idea, I've updated the sentence.